src/services/importService.ts

```diff
@@ -96,6 +128,30 @@ export default class ImportService implements IImportService {
         return generatedAssetMetadata;
     }
 
+    /**
+     * Generate parent asset based on V1 Project video assets
+     * @param project - V1 Project Content and File Information
+     */
+    public async createParentVideoAsset(v1Project: IFileInfo): Promise<IAsset> {
+        let parentAsset: IAsset;
+        // parentAsset = AssetService.createAssetFromFilePath(v1Project.file.path);
+        parentAsset = {
+            format: "mp4",
+            id: new MD5().update(v1Project.file.path.replace(/\.[^/.]+$/, "")).digest("hex"),
+            name: v1Project.file.path.replace(/\.[^/.]+$/, "").replace(/^.*[\\\/]/, ""),
+            path: `file:${v1Project.file.path.replace(/\.[^/.]+$/, "")}`,
+            size: {
```

Comment:
We should be pulling actual width/height values if available in the source project.